source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,129 @@
 See the [changelog in progress](https://bit.ly/2nK3cVf) for the upcoming release. See the [Legacy Self-Hosted Mattermost Changelog](legacy-self-hosted-changelog) for details on all Mattermost self-hosted releases prior to v5.37. 
 
 Latest Mattermost Releases:
+- [Release v7.0 - Feature Release](#release-v7-0-feature-release)
 - [Release v6.7 - Feature Release](#release-v6-7-feature-release)
 - [Release v6.6 - Feature Release](#release-v6-6-feature-release)
 - [Release v6.5 - Feature Release](#release-v6-5-feature-release)
-- [Release v6.4 - Feature Release](#release-v6-4-feature-release)
 - [Release v6.3 - Extended Support Release](#release-v6-3-extended-support-release)
 
+## Release v7.0 - [Feature Release](https://docs.mattermost.com/upgrade/release-definitions.html#feature-release)
+
+**Release Day: 2022-06-15**
+
+### Important Upgrade Notes
+ - MySQL self-hosted customers may notice the migration taking longer than usual when having a large number of rows in the ``FileInfo`` table. For MySQL, it takes around 19s for a table of size 0.7M rows. For PostgreSQL, it takes around 1.3ms for a table of size 0.7M rows.
+ - CRT GA upgrade notes?
+ - All channel header icons registered by plugins were moved to the new App Bar, even if they do not explicitly use the new registry function to render a component there.
+ - Boards: manual refresh on license change
+ - The value of ``ServiceSettings.TrustedProxyIPHeader`` defaults to empty from now on. A previous bug prevented this from happening in certain conditions. Customers are requested to check for these values in their config and set them to nil if necessary.
+ - The **System Console** session lengths configurations settings were changed to use a new hours unit, replacing the days. If anything goes awry during a server upgrade, it could result in users not being able to log in. For customer who use environment variables for those config settings, the automatic migration will not work and they'll need to add new environment variables: ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS=``, ``MM_SERVICESETTINGS_SESSIONLENGTHMOBILEINHOURS=``, and ``MM_SERVICESETTINGS_SESSIONLENGTHSSOINHOURS=``. The values need to be 24x the existing values in days. For example, if ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINDAYS=30``, they should set ``MM_SERVICESETTINGS_SESSIONLENGTHWEBINHOURS=720``.
```

Comment:
@mkraft @laneycs @wiersgallak I want to flag this note, it seems concerning. Session lengths are a significant and consistent source of negative NPS feedback. I'm wondering:

1. What's the chance users will be logged out or unable to log-in after this change?
2. What portion of customers are using environment variables and will manually have to increase session lengths 24x? If they don't do it (which I think is unfortunately very likely), a 24x lower session length is going to cause a lot of users serious pain and frustrations due to frequent logouts (many of whom will likely have trouble logging back in)

Are there other ways we can implement it without the potential for breaking changes?

Reply:
@esethna The CS and support teams (cc @svelle @sadohert) are aware of this change and are planning to communicate it as well. I don't think it's unreasonable what we're expecting: customers read the release notes and take required action. I agree that likely some administrator of an instance using environment variables for session lengths may miss the release note and get logged out at a shorter interval (ex. every 1h15m if their sessions were set to 30 days), but I'm OK with attempting this strategy, gathering feedback, and iterating on it to come up with a more intricate future strategy if necessary.